### ISSUE TITLE
The custom twig error page is not working.

### ISSUE THREAD

**Issue by seehat** [fvsch/kirby-twig#8]
It seams that the path when loading the error template isn't correct. Shouldn't it be like this:

``` php
$html = Tpl::load(dirname(__DIR__) . DS . 'templates' . DS . 'twigerror.php', [
    'title' => $title,
    'message' => $message,
    'file' => $file,
    'code' => $code
]);
return new Response($html, 'html', 500);
```

**Comment by fvsch** [comment]
Welp, that's embarrassing.

**Comment by fvsch** [comment]
Fixed in 2.0.1. Thanks for the heads up!
